### PR TITLE
Changing to UI-based namespace creation flow

### PR DESCRIPTION
**Pull request by ousleyp**
cc @alexxa @nellyc @rthallisey 

Updating the namespace creation flow for CNV 2.1.

(for enterprise-4.2 and 4.3)

### CODE REVIEW

File: modules/cnv-preparing-to-install.adoc [hunk 1]
```diff
@@ -14,9 +14,11 @@ Before deploying {CNVProductName}, create a namespace that is named
...
 .Procedure
 
-* Create the `openshift-cnv` namespace by running the following
-command:
-+
-----
-$ oc create ns openshift-cnv
-----
\ No newline at end of file
+* From the {product-title} web console, navigate to the
```
Comment (bergerhoffer) [inline modules/cnv-preparing-to-install.adoc]: These should be numbered bullets since there is more than one step now.

**Review by bergerhoffer — COMMENTED** [review]
Just one thing, and then a question on the version(s) this applies for. Says only 4.2, but should this also be carried forward for 4.3?

If it is for 4.2 only, then it should be submitted directly to enterprise-4.2 (and not master, because master is the latest and greatest, like 4.3)

**Comment by alexxa** [comment]
@bergerhoffer I would merge it to 4.2 only. There is a BZ on OCP which is supposed to be fixed in 4.3 release and then creating NS procedure will be removed completely from CNV docs.

**Comment by ousleyp** [comment]
> @bergerhoffer I would merge it to 4.2 only. There is a BZ on OCP which is supposed to be fixed in 4.3 release and then creating NS procedure will be removed completely from CNV docs.

@alexxa, actually I just realized it should be for both 4.2 and 4.3 to align with the OCP docs process. We can still definitely remove it for 4.3, so don't worry about that. :)

**Pull request merged**